weave-js/src/core/ops/domain/gql.ts

```diff
@@ -1120,6 +1120,15 @@ export const toGqlField = (
     return gqlBasicField('isGenerated');
   } else if (forwardOp.op.name === 'artifactVersion-isLinkedToGlobalRegistry') {
     return gqlBasicField('isLinkedToGlobalRegistry');
+  } else if (forwardOp.op.name === 'artifactVersion-tags') {
+    return [
+      gqlObjectField(forwardGraph, forwardOp, 'tags', undefined, {
+        extraFields: gqlBasicField('id')
+          .concat(gqlBasicField('name'))
+          .concat(gqlBasicField('tagCategoryName'))
+          .concat(gqlBasicField('attributes')),
+      }),
+    ];
```

Comment:
note: largely adapted from the existing `artifact-rawTags` op for artifact *collection* tags:

https://github.com/wandb/weave/blob/69710f9342db3b0fbb3f46ad64b3d380296ef71c/weave-js/src/core/ops/domain/gql.ts#L807-L826